### PR TITLE
Add SSL guide page

### PR DESCRIPTION
**Pull request by ianmalott**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _guide/0801-ssl-tls.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: SSL/TLS
+permalink: /guide/ssl-tls/
+phase: run
+---
+
+SSL is a protocol that provides for the encryption of network traffic to and from your Balancers.
+
+TLS is the successor to SSL.
```
Comment (nzoschke) [inline _guide/0801-ssl-tls.md]: This can join the above sentence.

File: _guide/0801-ssl-tls.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<span class="diff-u">      - 80:8000</span>
+<span class="diff-a">      - 443:8000</span>
+<span class="diff-u">  worker:</span>
+<span class="diff-u">    build: .</span>
+<span class="diff-u">    command: ["node", "worker.js"]</span>
+<span class="diff-u">    environment:</span>
+<span class="diff-u">      - GITHUB_API_TOKEN</span>
+</pre>
+
+The Balancer will now listen for traffic from the internet on port 443, secure connections with the HTTPS protocol using a self-signed cert, and forward requests to app Containers that are bound to port 8000.
```
Comment (nzoschke) [inline _guide/0801-ssl-tls.md]: to the app web Containers that are bound...

File: _guide/0801-ssl-tls.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Run `convox doctor` to validate your port protocol label and Balancer definitions:
+
+<pre class="terminal">
+<span class="command">convox doctor</span>
+
+### Run: Balancer
```
Comment (nzoschke) [inline _guide/0801-ssl-tls.md]: ### Run: SSL/TLS
[<span class="pass">✓</span>] Application secured with SSL/TLS
[<span class="pass">✓</span>] Service <span class="service">web</span> port 443 responds to HTTPS

File: _guide/0801-ssl-tls.md (hunk 1 above)
Comment (ianmalott) [inline _guide/0801-ssl-tls.md]: Deleting entirely as part of the SSL/TLS -> SSL change.

File: _guide/0801-ssl-tls.md (hunk 2 above)
Comment (ianmalott) [inline _guide/0801-ssl-tls.md]: going with

> to the `web` Service processes that are bound

in keeping with the conversation earlier today about "process" over "container" and a process being an instance of a Service

File: _guide/0801-ssl-tls.md (hunk 3 above)
Comment (ianmalott) [inline _guide/0801-ssl-tls.md]: ✅

File: _guide/11-links.md [hunk 4]
```diff
@@ -17,23 +17,26 @@ A Link is defined in the `links:` and `environment:` sections of `docker-compose
...
+<span class="diff-u">      - 443:8000</span>
 <span class="diff-u">  worker:</span>
 <span class="diff-u">    build: .</span>
 <span class="diff-u">    command: ["node", "worker.js"]</span>
 <span class="diff-u">    environment:</span>
-<span class="diff-u">     - GITHUB_API_TOKEN</span>
-<span class="diff-a">     - REDIS_URL</span>
-<span class="diff-a">  links:</span>
-<span class="diff-a">    - redis</span>
+<span class="diff-u">      - GITHUB_API_TOKEN</span>
```
Comment (nzoschke) [inline _guide/11-links.md]: is this whitespace correct?

Comment (nzoschke) [inline _guide/11-links.md]: It looks correct locally. Guess it's something strange in the github diff viewer

**Pull request merged**